python/nav/web/templates/navlets/_add_navlet_modal.html

```diff
@@ -3,7 +3,11 @@ <h4>Add a widget to your page</h4>
     {% for navlet in navlets %}
       <li>
         <div class="panel">
-          <form class="add-user-navlet right" action="{% url 'add-user-navlet' dashboard_id %}" method="post">
+          <form
+                  class="add-user-navlet right"
+                  hx-post="{% url 'add-user-navlet' dashboard_id %}"
+                  hx-swap="beforeend"
+          >
```

Comment:
What happens if you keep `action` and `post` in addition to what's here now?

Reply:
Is progressive fallback not a goal here? (not that I'm sure this ever had functional progressive fallback).